docs/integrations/credentials/httpRequest.md

```diff
@@ -6,7 +6,17 @@ You can use these credentials to authenticate the following nodes:
 
 ## Prerequisites
 
-You must use the authentication method required by the app or service you want to query. The following authentication methods are available:
+You must use the authentication method required by the app or service you want to query.
+
+### Existing credential types
+
+n8n recommends uing this option whenever it's available.  It offers an easier way to set up and manage credentials, compared to configuring generic credentials.
```

Comment:
n8n recommends using this option whenever there's a credential type already available for the service you want to connect to

(PS: Do we not have tooling set up to catch things like 'uing' by this stage?)